[2022-09-05 03:07:31,812] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.thrid_task 2022-08-06T00:00:00+00:00 [queued]>
[2022-09-05 03:07:31,979] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.thrid_task 2022-08-06T00:00:00+00:00 [queued]>
[2022-09-05 03:07:31,980] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-09-05 03:07:31,981] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2022-09-05 03:07:31,981] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-09-05 03:07:32,109] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): thrid_task> on 2022-08-06T00:00:00+00:00
[2022-09-05 03:07:32,123] {standard_task_runner.py:52} INFO - Started process 256 to run task
[2022-09-05 03:07:32,197] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v2', 'thrid_task', '2022-08-06T00:00:00+00:00', '--job-id', '29', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpav6fl1_2', '--error-file', '/tmp/tmpn2d5g9s_']
[2022-09-05 03:07:32,214] {standard_task_runner.py:77} INFO - Job 29: Subtask thrid_task
[2022-09-05 03:07:32,648] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v2.thrid_task 2022-08-06T00:00:00+00:00 [running]> on host 2b8e628fc16c
[2022-09-05 03:07:33,239] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v2
AIRFLOW_CTX_TASK_ID=thrid_task
AIRFLOW_CTX_EXECUTION_DATE=2022-08-06T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-06T00:00:00+00:00
[2022-09-05 03:07:33,241] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2022-09-05 03:07:33,242] {bash.py:158} INFO - Running command: echo hey, I am task3 and will be running after task1 at the same time as task2!
[2022-09-05 03:07:33,305] {bash.py:169} INFO - Output:
[2022-09-05 03:07:33,305] {bash.py:173} INFO - hey, I am task3 and will be running after task1 at the same time as task2!
[2022-09-05 03:07:33,311] {bash.py:177} INFO - Command exited with return code 0
[2022-09-05 03:07:33,642] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=thrid_task, execution_date=20220806T000000, start_date=20220905T030731, end_date=20220905T030733
[2022-09-05 03:07:33,889] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-09-05 03:07:33,939] {local_task_job.py:146} INFO - Task exited with return code 0
